stdlib/string.ncl

```diff
@@ -15,19 +15,20 @@
           error
       ```
       "%
-    = fun l s =>
-      if %typeof% s == `String then
-        if s == "true" || s == "True" then
-          "true"
-        else if s == "false" || s == "False" then
-          "false"
+      = fun l s =>
+        if %typeof% s == `String then
+          if s == "true" || s == "True" then
+            "true"
+          else
```

Comment:
I think this is the last substantial pet-peeve of mine with the current formatting: it adds one level of nesting for a chain of `if..else..if..else..if...else`. Would it be doable in topiary to have a rule that if an else has an if-then-else as a children, we remove the first newline (between the else and the inner if) and remove one level of indentation, so that we get the more idiomatic:

```nickel
if ... then
  ...
else if .. then
 ...
else if .. then
 ...
else
  ...
```

Reply:
I'm going to say "maybe"! Adding exceptions seem to fall into two categories: really easy, or almost impossible :sweat_smile:

I'll see what I can do :muscle:

Reply:
Issue: https://github.com/tweag/topiary/issues/399

Reply:
Fixed by https://github.com/tweag/topiary/pull/402